aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: kubernetes-dashboard
 version: 2.0.2
-appVersion: 2.0.2
+appVersion: 2.0.3
```

Comment:
@floreks @maciaszczykm c'mon, why again? 😣

Reply:
I have updated version and pushed to the `gh-pages` https://github.com/kubernetes/dashboard/tree/gh-pages